code/datums/supplypacks/hospitality.dm

```diff
@@ -63,7 +63,8 @@
 			/obj/item/pizzabox/mushroom,
 			/obj/item/pizzabox/meat,
 			/obj/item/pizzabox/vegetable,
-			/obj/item/pizzabox/pineapple
+			/obj/item/pizzabox/pineapple,
+			/obj/item/weapon/material/knife/plastic
```

Comment:
Unfortunately, because this crate has `	num_contained = 5` defined, it picks [5] elements from this list at random. Which, with this change, _could_ include 1 knife. Or 5 knives and no pizza.

Reply:
Hnngh.